src/schema/objects/columns.yaml

```
@@ -162,6 +162,37 @@ duration:
   type: number
   unit: s
   minimum: 0
+x_coordinate:
+  name: x_coordinate
+  display_name: Gaze x-coordinate
+  description: |
+    Gaze position x-coordinate of the recorded eye, in the coordinate
+    units specified in the corresponding metadata sidecar.
+  type: number
+y_coordinate:
+  name: y_coordinate
+  display_name: Gaze y-coordinate
+  description: |
+    Gaze position y-coordinate of the recorded eye, in the coordinate
+    units specified in the corresponding metadata sidecar.
+  type: number
+pupil_size:
+  name: pupil_size
+  display_name: Pupil size
+  description: |
+    Pupil size or area of the recorded eye, in the
+    units specified in the corresponding metadata sidecar.
+    It is RECOMMENDED to indicate the particular type of pupil size that is
+    being recorded in the `Description` field of this column.
+    It is RECOMMENDED to specify the `Units` field corresponding to this column.
+  type: number
+et_timestamp:
```

The usual convention is `<name>[__<context>]`

```suggestion
timestamp__eyetrack:
```